beesdoo_product/models/beesdoo_product.py

```diff
@@ -27,6 +27,13 @@ class BeesdooProduct(models.Model):
         "beesdoo.product.label", domain=[("type", "=", "delivery")]
     )
 
+    fds_label = fields.Many2one(
+        "beesdoo.product.hazard", string="FDS label", domain=[("type", "=", "fds")], translate=True, default=lambda self: self.env['beesdoo.product.hazard'].search([["type", "=", "fds"],["name", "=", "Not required"]])
+    )
+    hazard_label = fields.Many2one(
+        "beesdoo.product.hazard", string="Hazard label", domain=[("type", "=", "hazard")], translate=True, default=lambda self: self.env['beesdoo.product.hazard'].search([["type", "=", "hazard"],["name", "=", "No"]])
+    )
+
```

Comment:
could you respect PEP8 and have one field attribute on each line

Reply:
Yes, absolutely: pretty code always works better :-)